tests/unit/backend/layers/business/test_business.py

```diff
@@ -215,7 +234,7 @@ def test_get_collection_version_for_unpublished_collection_ok(self):
 
         fetched_version = self.business_logic.get_collection_version(version.version_id)
 
-        self.assertIsNotNone(fetched_version.published_at)
+        self.assertIsNone(fetched_version.published_at)
```

Comment:
Typo in the original test. An unpublished collection needs to have `published_at` set as None.